language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -253,7 +255,7 @@ genDefDataType curPkgId conName mod tpls def =
                             ["daml.registerTemplate(" <> conName <> ");"]
                         refs = Set.unions (fieldRefs ++ argRefs)
                     in
-                    ((makeType typeDesc, dict ++ registrations), refs)
+                    ((makeType ["//" <> conName] ++ typeDesc, dict ++ registrations), refs)
```

Comment:
What is this about? Are we adding a comment here? I'm not convinced adding comments to generated code has value.

Reply:
Oops. Debug detritus. Will remove!